### ISSUE TITLE
Installation of 4.3.7 ME at All-Inkl not possible

### ISSUE THREAD

**Issue by jscholtysik** [contao/managed-edition#6]
My system:

Provider: All- Inkl Premium
PHP: 5.6.29-nmm1 (cli)

I made the following steps at All-Inkl:

1. Created a subdomain: contao4.domain.de

2. Installed Composer in the root directory with:

`curl -sS https://getcomposer.org/installer | php`

3. Installed Contao 4.3.7 with:

`php composer.phar create-project contao/managed-edition contao`

4. Changed the DocumentRoot to /contao/web:

![bildschirmfoto 2017-04-19 um 15 13 32](https://cloud.githubusercontent.com/assets/1425566/25181745/d3b55cae-2512-11e7-83c7-abaef66c32e0.png)

The install.php is not reachable under sub.domain.de/install.php and the following error message appears:

> Fatal error: Uncaught exception 'Symfony\Component\Filesystem\Exception\IOException' with message 'Using the Contao\ClassLoader class has been deprecated and will no longer work in Contao 5.0. Use the Composer autoloader instead.' in /www/htdocs/w016bce8/contao/vendor/symfony/symfony/src/Symfony/Component/Filesystem/LockHandler.php:86 Stack trace: #0 /www/htdocs/w016bce8/contao/vendor/contao/core-bundle/src/Command/AbstractLockedCommand.php(32): Symfony\Component\Filesystem\LockHandler->lock() #1 /www/htdocs/w016bce8/contao/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php(265): Contao\CoreBundle\Command\AbstractLockedCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\BufferedOutput)) #2 /www/htdocs/w016bce8/contao/vendor/contao/installation-bundle/src/Controller/InstallationController.php(170): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\BufferedOutp in /www/htdocs/w016bce8/contao/vendor/symfony/symfony/src/Symfony/Component/Filesystem/LockHandler.php on line 86

**Comment by leofeyer** [comment]
Can you try `sub.domain.de/contao/install` instead, please?

**Comment by jscholtysik** [comment]
Using this path the following error is in /var/logs/prod.log:

> [2017-04-19 15:30:50] request.INFO: Matched route "contao_install". {"route":"contao_install","route_parameters":{"_scope":"backend","_token_check":true,"_controller":"Contao\\InstallationBundle\\Controller\\InstallationController::installAction","_route":"contao_install"},"request_uri":"http://contao4.bitsandmedia.de/contao/install","method":"GET"} []
[2017-04-19 15:30:50] app.CRITICAL: An exception occurred. {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Using the Contao\\RequestToken class has been deprecated and will no longer work in Contao 5.0. Use the Symfony CSRF service via the container instead. at /www/htdocs/w016bce8/contao/vendor/symfony/symfony/src/Symfony/Component/Filesystem/LockHandler.php:86)"} []

**Comment by leofeyer** [comment]
Does the issue also occur if you download and install the .zip package from contao.org instead of using `composer create-project`?

**Comment by jscholtysik** [comment]
Yes, same error.

**Comment by NinaG** [comment]
Did you try it with PHP 7 instead? I successfully installed C4 at All-Inkl some days ago - but I used PHP 7.

**Comment by jscholtysik** [comment]
No, I haven't tried it yet, but I think Contao 4.3 ME should run also on PHP 5.6 without any problems, because the system requirements specify a minimum of PHP 5.5.

**Comment by ausi** [comment]
Could you please try if the [Contao check](https://github.com/contao/check) finds any problems with your webspace?

**Comment by jscholtysik** [comment]
I checked my webspace, no error appears in the Contao check 12.0:

![bildschirmfoto 2017-04-20 um 13 30 35](https://cloud.githubusercontent.com/assets/1425566/25228711/c3236e56-25cd-11e7-83be-e108f0e66853.png)

**Comment by fritzmg** [comment]
I don't think this is something the Check would be able to detect. It appears the `E_USER_DEPRECATED` errors are thrown and then catched by Symfony's `LockHandler` for some reason and then thrown again as `IOException`s.

**Comment by ausi** [comment]
I think the problem is that in [LockHandler.php:71-87](https://github.com/symfony/symfony/blob/9c0067b19f2e9d6a31565a10f41c826994e388ae/src/Symfony/Component/Filesystem/LockHandler.php#L71-L87) the `fopen()` call fails but `error_get_last()` doesn’t get the right error message.

**Comment by jscholtysik** [comment]
Today I tried my installation of Contao 4 at All-Inkl once more with PHP 7.1 CGI/FPM: No problem and the install tool start page appeared. 
Then I tried PHP 5.5 CGI/FPM, also no problem and also here the install tool start page appeared. Then I tried again PHP 5.6 CGI/FPM. Now also this configuration worked flawlessly and also here the install tool worked... 
Very strange and without any changes of the configuration with PHP 5.6 CGI/FPM

Sorry and thanks for your efforts!
